src/main/java/core/basesyntax/MainApp.java

```diff
@@ -1,5 +1,19 @@
 package core.basesyntax;
 
 public class MainApp {
+    public static void main(String[] machine) {
+        Machine truck = new Truck();
+        Machine bulldozer = new Bulldozer();
+        Machine excavator = new Excavator();
+        truck.doWork();
+        bulldozer.stopWork();
```

Comment:
According to the checklist item 'Write informative messages in methods', the messages should indicate what type of `Machine` is working. Make sure that the `doWork` and `stopWork` methods in `Truck`, `Bulldozer`, and `Excavator` classes print informative messages indicating the machine type.